code-examples/missingNumber/src/MissingNumberTest.java

```diff
@@ -0,0 +1,38 @@
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+class MissingNumberTest {
+
+    private MissingNumber mn = new MissingNumber();
+
+    @Test
+    void canCreateArrayWithMissingInteger7() {
+        int[] arrayMissing7 = arr();
+        Assert.assertEquals("[1, 2, 3, 4, 5, 6, 8, 9, 10]", Arrays.toString(arrayMissing7));
+    }
+
+    private int[] arr() {
+        return mn.createArray(10, 7);
+    }
+
+    @Test
+    void checkLR() {
```

Comment:
This should be renamed to `givenMissing7InArray_returns7`, I refactored this test until I got the algorithm that I wanted and didn't keep renaming it as I progressed, so the name made more sense when I first created this method.